dnsimple/registrar_delegation.go

```diff
@@ -7,12 +7,28 @@ import (
 // Domain represents a list of name servers that correspond to a domain delegation.
 type Delegation []string
 
+// Vanity Name Server
+type VanityNameServer struct {
```

Comment:
@weppos Yes, the difference here is that for Name Servers we just have an array of strings, but for Vanity Name Servers we return a more complex payload.